vowpalwabbit/lda_core.cc

```diff
@@ -180,6 +180,14 @@ inline float fastdigamma(float x)
 #include <smmintrin.h>
 #endif
 
+#if defined(__ARM_NEON__) || defined(__ARM_NEON)
+#include <arm_neon.h>
```

Comment:
I'm guessing this file is present on your Macbook M1 when using clang? 

More generally, we are going to need to improve our instructions for building VW. AFAIK, our current process relies on having engineers with old and new Macbooks, and we build the ubuntu versions on EC2 instances.

Reply:
> I'm guessing this file is present on your Macbook M1 when using clang?

Yes, also when compiling on graviton instances. 

> More generally, we are going to need to improve our instructions for building VW. AFAIK, our current process relies on having engineers with old and new Macbooks, and we build the ubuntu versions on EC2 instances.

It would be great if we can do that with Github Action I am adding a story in the icebox.